From e5315da1e748dc541d50796fb05233da65e86b6b Mon Sep 17 00:00:00 2001
From: Dennis Heimbigner <dmh@ucar.edu>
Date: Sat, 4 May 2019 12:29:42 -0600
Subject: [PATCH] Option --disable-byterange fails if libcurl is not found

re: https://github.com/Unidata/netcdf-c/issues/1390

Change configure.ac to properly test for
combination of no-curl and disable-byterange.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6f888c13a..4b57f6cf9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -885,7 +885,7 @@ AC_ARG_ENABLE([byterange],
 test "x$enable_byterange" = xyes || enable_byterange=no
 AC_MSG_RESULT($enable_byterange)
 # Need curl for byte ranges
-if test "x$found_curl" = xno ; then
+if test "x$found_curl" = xno && test "x$enable_byterange" = xyes ; then
   AC_MSG_ERROR([curl required for byte range support. Install curl or build without --enable-byterange.])
   enable_byterange=no
 fi
